docs/api/python-api.md

```
@@ -45,6 +45,43 @@ print(f"Columns: {table.column_names}")
 print(f"Geometry column: {table.geometry_column}")
 ```
 
+### S3-Compatible Storage
+
+`read_partition()` supports custom S3 endpoints for reading from MinIO, Cloudflare R2, source.coop, and other S3-compatible services:
+
+```python
```

markdownlint-cli2 (0.22.1)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)